app/frontend/src/components/admin/Clients.vue

```diff
@@ -76,19 +122,17 @@ export default {
       keycloakService
         .getServiceClients()
         .then(response => {
-          // reformat data to show in our data table of service clients
-          const reduced = response.data.reduce((a, b) => {
-            if (!a[b.clientId]) a[b.clientId] = []; //If this type wasn't previously stored
-            a[b.clientId].push(b.environment);
-            return a;
-          }, {});
-
-          const clients = Object.keys(reduced).map(k => {
+          const data = response.data;
+          const clients = Object.keys(data).map(k => {
+            let client = data[k];
             return {
-              name: k,
-              dev: reduced[k].includes('dev'),
-              test: reduced[k].includes('test'),
-              prod: reduced[k].includes('prod')
+              acronym: client.acronym,
```

Comment:
i've changeed the structure of the data from the backend so this mapping to data table is different